src/commands/cmd_bloom_filter.cc

```diff
@@ -104,14 +105,86 @@ class CommandBFExists : public Commander {
     redis::BloomChain bloom_db(svr->storage, conn->GetNamespace());
     int ret = 0;
     auto s = bloom_db.Exist(args_[1], args_[2], &ret);
+    if (s.IsNotFound()) return {Status::RedisExecErr, "key is not found"};
     if (!s.ok()) return {Status::RedisExecErr, s.ToString()};
 
     *output = redis::Integer(ret);
     return Status::OK();
   }
 };
 
+class CommandBFInfo : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 2);
+    while (parser.Good()) {
+      if (parser.EatEqICase("capacity")) {
+        type_ = BloomInfoType::kCapacity;
+      } else if (parser.EatEqICase("size")) {
+        type_ = BloomInfoType::kSize;
+      } else if (parser.EatEqICase("filters")) {
+        type_ = BloomInfoType::kFilters;
+      } else if (parser.EatEqICase("items")) {
+        type_ = BloomInfoType::kItems;
+      } else if (parser.EatEqICase("expansion")) {
+        type_ = BloomInfoType::kExpansion;
+      } else {
+        return {Status::RedisParseErr, "Invalid info argument"};
+      }
+    }
+
+    return Commander::Parse(args);
+  }
+
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    redis::BloomChain bloom_db(svr->storage, conn->GetNamespace());
+    BloomFilterInfo info;
+    auto s = bloom_db.Info(args_[1], &info);
+    if (s.IsNotFound()) return {Status::RedisExecErr, "key is not found"};
+    if (!s.ok()) return {Status::RedisExecErr, s.ToString()};
+
+    switch (type_) {
+      case BloomInfoType::kAll:
+        *output = redis::MultiLen(2 * 5);
+        *output += redis::SimpleString("Capacity");
+        *output += redis::Integer(info.capacity);
+        *output += redis::SimpleString("Size");
+        *output += redis::Integer(info.bloom_bytes);
+        *output += redis::SimpleString("Number of filters");
+        *output += redis::Integer(info.n_filters);
+        *output += redis::SimpleString("Number of items inserted");
+        *output += redis::Integer(info.size);
+        *output += redis::SimpleString("Expansion rate");
+        *output += redis::Integer(info.expansion);
```

Comment:
https://github.com/RedisBloom/RedisBloom/blob/master/src/rebloom.c#L990

Since `info.expansion` might be null if it's 0?